README.md

```diff
@@ -1,53 +1,70 @@
+friendly. This allows commercial applications to be written that use the open
+source OpenMRN libraries without having to be open source themselves.
 
 # Getting Started
 
-Most of the documentation for OpenMRN is in [doxygen](http://www.stack.nl/~dimitri/doxygen/) format. The best way to view the documents is to
-build the HTML files (instructions below).
+Most of the documentation for OpenMRN is in
+[doxygen](http://www.stack.nl/~dimitri/doxygen/) format. The best way to view
+the documents is to build the HTML files (instructions below).
 
-The existing OpenMRN stack makes heavy use of Linux build tools. As a result, the best way to get up and running, and to build the documentation,
-is within Linux or Mac OS X. If you have a Windows, we recommend that you create a Linux virtual machine. Below are some high-level
-instructions on getting started.
+The existing OpenMRN stack makes heavy use of Linux build tools. As a result,
+the best way to get up and running, and to build the documentation, is within
+Linux or Mac OS X. If you have a Windows, we recommend that you create a Linux
+virtual machine. Below are some high-level instructions on getting started.
 
 ## Create a Linux Virtual Machine
 
-* Install Oracle [VirtualBox](https://www.virtualbox.org/) with is available for free.
+* Install Oracle [VirtualBox](https://www.virtualbox.org/) with is available for
+free.
 * Create a virtual machine
     * Ideally, this should be a 64-bit virtual machine
     * Assign it at least 4GB of RAM
     * Assign it multiple cores (improves build speed)
-    * Create a _dynamically allocated_ virtual disk that is larger than the default. The default is 10GB, and we recommend at least 30GB. More is
-    better because the virtual machine file itself will expand, but the maximum size is hard to change in the future
-* Download a copy of Ubuntu desktop. At the time of this writing, 16.04 LTS is a good machine. Download this as an ISO disk image
-* Start the virtual machine. It will ask you to select a startup disk. Select the ISO file that you downloaded for Ubuntu desktop
+    * Create a _dynamically allocated_ virtual disk that is larger than the
+      default. The default is 10GB, and we recommend at least 30GB. More is
+      better because the virtual machine file itself will expand, but the
+      maximum size is hard to change in the future
+* Download a copy of Ubuntu desktop. At the time of this writing, 16.04 LTS is a
+  good machine. Download this as an ISO disk image
+* Start the virtual machine. It will ask you to select a startup disk. Select
+  the ISO file that you downloaded for Ubuntu desktop
 
-Once you've done this, you'll need to do a few more things before you can get the source code and build the documentation.
+Once you've done this, you'll need to do a few more things before you can get
+the source code and build the documentation.
 
 ## Installing Required Software (Linux or Mac)
```

Comment:
Consider adding to the title that this is how to build the documentation. Or maybe as another title a bit below.